Buildings/Fluid/Air/BaseClasses/ElectricHeater.mo

```diff
@@ -0,0 +1,144 @@
+within Buildings.Fluid.Air.BaseClasses;
+model ElectricHeater "Model for electric heater"
+  //extends Buildings.Fluid.HeatExchangers.Heater_T;
+  extends Buildings.Fluid.Interfaces.PartialTwoPortInterface;
+  extends Buildings.Fluid.Interfaces.TwoPortFlowResistanceParameters(
+    final computeFlowResistance=(abs(dp_nominal) > Modelica.Constants.eps));
+
+  parameter Real eff "Effciency of electrical heater";
```

Comment:
Use `Modelica.SIunits.Efficiency` and call it `eta` (for consistency with other models).